apps/server/src/modules/lesson/controller/dto/lesson.response.ts

```diff
@@ -39,7 +39,8 @@ export class LessonResponse {
 		// @deprecated _id used in legacy client
 		this._id = lesson.id;
 		this.name = lesson.name;
-		this.courseId = lesson.course.id;
+		this.courseId = lesson?.course?.id;
```

Comment:
why does it need `lesson?` instead of `lesson.course.id` and `lesson.courseGroup?.id` ?
in my understanding the lesson is always present and a lesson always has a course, right? otherwise (without a course) I would not know what to do with a lesson instead

Reply:
fixed